content/reference/storage/limits.md

```diff
@@ -0,0 +1,25 @@
+---
+title: 'Limits'
+weight: 80
+---
+
+Workers KV supports:
+
+- Up to 100 Namespaces per account
+- Unlimited keys per Namespace
+- Keys of up to 512 bytes
+- Values of up to 10 MB
+- Unlimited reads per second per key
+- Up to one write per second per key
+
+If your application requires higher limits, please [let us know](https://support.cloudflare.com).
```

Comment:
Just checking, this is a task support is used to enabling? 

I don't see support recommending to contact them in [their article](https://support.cloudflare.com/hc/en-us/articles/360001657552-Billing-for-Cloudflare-Workers-and-Workers-KV) so we should be sure this is where we should direct them

Reply:
This text was here previously, so I'm not sure to be honest.